eng/Version.Details.xml

```diff
@@ -1,58 +1,58 @@
+    <Dependency Name="System.Text.Encodings.Web" Version="5.0.0-preview.3.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0-preview.3-runtime.20179.1" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0-preview.3-runtime.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="5.0.0-preview.3.20179.1" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="5.0.0-preview.3.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="5.0.0-preview.3.20179.1" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="5.0.0-preview.3.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
     <!--
       Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
       All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.0-preview.3.20179.1" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.0-preview.3.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
     <!-- Listed as a dependency to workaround https://github.com/dotnet/cli/issues/10528 -->
-    <Dependency Name="Microsoft.NETCore.Platforms" Version="5.0.0-preview.3.20179.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="5.0.0-preview.3.20179.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5eda36ed557d789d888647745782b261472b9fa3</Sha>
     </Dependency>
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="5.0.0-beta.20171.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>7dbc907fa03eacf4c57f827cb4235d77b7ed4fcd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-3.20170.19" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-3.20170.19" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
dotnet/runtime doesn't ingest roslyn - we'll need to add a sub from roslyn to this repo

Reply:
If runtime doesn't ingest roslyn then do we need the CPD?

Reply:
We don't

Reply:
removed